api/src/test/java/uk/gov/dhsc/htbhf/eligibility/service/EligibilityStatusCalculatorTest.java

```diff
@@ -0,0 +1,41 @@
+package uk.gov.dhsc.htbhf.eligibility.service;
+
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+import uk.gov.dhsc.htbhf.eligibility.model.EligibilityStatus;
+import uk.gov.dhsc.htbhf.eligibility.model.dwp.DWPEligibilityResponse;
+import uk.gov.dhsc.htbhf.eligibility.model.hmrc.HMRCEligibilityResponse;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static uk.gov.dhsc.htbhf.eligibility.helper.DWPEligibilityResponseTestFactory.aDWPEligibilityResponseWithStatus;
+import static uk.gov.dhsc.htbhf.eligibility.helper.HMRCEligibilityResponseTestFactory.anHMRCEligibilityResponseWithStatus;
+
+public class EligibilityStatusCalculatorTest {
+
+    private final EligibilityStatusCalculator statusCalculator = new EligibilityStatusCalculator();
+
+    @ParameterizedTest(name = "Should return {2} when DWP status is {0} and HMRC status is {1}")
+    @CsvSource({
+            "ELIGIBLE, ELIGIBLE, ELIGIBLE",
```

Comment:
Is it worthwhile adding all permutations in here so we're 100% covered? There's not too many more than you've got here already.

Reply:
snap :)

Reply:
Done.